python/ray/util/client/server/server.py

```diff
@@ -371,6 +421,21 @@ def _convert_args(self, arg_list, kwarg_map):
             kwargout[k] = convert_from_arg(kwarg_map[k], self)
         return argout, kwargout
 
+    def _prepare_runtime_env(self, job_runtime_env) -> List[str]:
+        """Download runtime environment to local node"""
+        missing_uris = []
+        uris = job_runtime_env.uris
+        from ray._private import runtime_env
+        with disable_client_hook():
+            for uri in uris:
+                try:
+                    runtime_env.fetch_package(uri)
+                    print("Adding!: ", runtime_env._get_local_path(uri))
+                    sys.path.insert(0, str(runtime_env._get_local_path(uri)))
+                except IOError:
+                    missing_uris.append(uri)
+        return missing_uris
```

Comment:
I don't think we need to return missing uris anymore, correct? Can we just let the exception propagate if this happens?